test/Microsoft.IdentityModel.Tests/IdentityUtilities.cs

```diff
@@ -93,6 +94,99 @@ public static JwtSecurityToken CreateJwtSecurityToken(string issuer, string audi
             return new JwtSecurityToken(header, payload, header.Base64UrlEncode(), payload.Base64UrlEncode(), "" );
         }
 
+        public static string CreateSamlSecurityToken(SecurityTokenDescriptor securityTokenDescriptor, SecurityTokenHandler tokenHandler)
```

Comment:
normally we drop the 'security' prefix for variables: securityTokenDescriptor -> tokenDescriptor. This matches what we do for SecurityTokenHandler.

Reply:
Fixed